pl_examples/pytorch_ecosystem/pytorch_geometric/README.md

```diff
@@ -0,0 +1,32 @@
+# Examples of [Pytorch Geometric](https://github.com/rusty1s/pytorch_geometric) with Lighting
+
+### Introduction
+
+PyTorch Geometric (PyG) is a geometric deep learning extension library for PyTorch. It relies on lower level libraries such as
+
+* PyTorch Cluster: A package consists of a small extension library of highly optimized graph cluster algorithms in Pytorch
+* PyTorch Sparse: A package consists of a small extension library of optimized sparse matrix operations with autograd support in Pytorch
+* PyTorch Scatter: A package consists of a small extension library of highly optimized sparse update (scatter and segment) operations for the use in PyTorch
+
+## Setup
+
+```
+pyenv install 3.7.8
+pyenv local 3.7.8
+python -m venv
+source .venv/bin/activate
+poetry install
+```
+
+## Current example lists
+
+| `DATASET` | `MODEL` | `TASK` | DATASET DESCRIPTION | MODEL DESCRIPTION                                                                                                                                                                   |                                                                                                                                                                     |
```

Comment:
@ananyahjha93 used another type of writing table wich turned to be much easier to read in the source
https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table